test/performance/benchmarks/broker-imc/continuous/200-broker-imc.yaml

```diff
@@ -128,7 +128,7 @@ spec:
                   mountPath: /var/secret
               terminationMessagePolicy: FallbackToLogsOnError
             - name: mako
-              image: gcr.io/knative-performance/mako-microservice:latest
+              image: gcr.io/knative-tests/test-infra/mako-microservice:latest
```

Comment:
Can you help me understand why this change is needed? It looks like Knative owns both of these registries. Why isn't the first one able to be centrally maintained?

Reply:
Sorry for the missing context.

When we run integration tests, we use the image `gcr.io/knative-tests/test-infra/prow-tests:stable` to create the Prow job pod, and upload the images built by `ko` to the registry of boskos projects. The images uploaded to boskos registry are considered as temporary so  we added a cleanup job to delete the images that are older than 30 days.

By moving the `mako-microservice` image to `knative-tests`, we will be able to add a similar cleanup job to delete the old images in `knative-performance` and `knative-eventing-performance` project.